doc/devel/coding_guide.rst

```diff
@@ -165,6 +165,48 @@ C/C++ extensions
 
 
 
+PR Review guidelines
+====================
+
+* We run on a 'use your judgement' system (for better or worse) and do
+  not have a strict nose-count rule.  If you have a commit bit, then
```

Comment:
@efiring what about reforming the labels so that they follow something like https://github.com/numpy/numpy/labels? I'm a huge fan of labels for quickly sorting out the things I can't do.